addons/stock/static/src/scss/stock_forecasted.scss

```diff
@@ -1,10 +1,10 @@
 .o_stock_forecasted_page {
     .o_report_replenishment {
         .o_grid_warning {
-            background-color: #f4cccc;
+            --table-bg: #{map-get($table-variants, 'warning')};
         }
         .o_grid_match {
-            background-color: #f0f0f0;
+            --table-bg: #{map-get($table-variants, 'light')};
         }
     }
```

Comment:
Adding the `table-warning` and  `table-light` classes makes this entire block of CSS  redundant, we can remove it.

Reply:
Indeed, I thought it was needed but these templates aren't stocked within the DB so it won't be applied, thanks 😅